src/turing_model.jl

```diff
@@ -200,11 +302,53 @@ function turing_model(
     if standardize
         μ_X, σ_X, X = standardize_predictors(X)
         μ_y, σ_y, y = standardize_predictors(y)
+        if !isnothing(Z)
+            #TODO: implement random-effects slope
+            throw(
+                ArgumentError(
+                    "TuringGLM currently does not support random-effects for slope terms"
+                ),
+            )
+        end
     end
 
     # Random-Effects Conditionals
     if has_ranef(formula)
-        # TODO
+        if priors isa DefaultPrior
+            custom_prior = CustomPrior(TDist(3), LocationScale(0, 2.5, TDist(3)), nothing)
+        else
+            custom_prior = priors
+        end
+        intercept_ranef = intercept_per_ranef(ranef(formula))
+        group_var = first(ranef(formula)).rhs
+        idx = get_idx(term(group_var), data)
+        # print for the user the idx
+        println("The idx are $(last(idx))\n")
+        @model function bernoulli_model_ranef(
```

Comment:
This code is three times duplicated which is asking for trouble. Is it possibly to write a neat abstraction to avoid the duplication?

Reply:
What code?
The `println`?

Reply:
The 40 or so lines are super similar